cpp/NINJA-steps.sh

```diff
@@ -225,12 +225,16 @@ compile_and_link() {
   local out=$4
   shift 4
 
+
   setglobal_compile_flags "$variant" "$more_cxx_flags" ""  # no dotd
 
   setglobal_link_flags $variant
 
   setglobal_cxx $compiler
 
+  echo ""
+  echo "Building: $cxx -o $out $flags $@ $link_flags"
+  echo ""
```

Comment:
this will probably make the build a lot noiser

Reply:
Oh yeah, I've been using it as a crutch for debugging builds.  I can comment it out before we merge this.